ksy_style_guide.adoc

```diff
@@ -353,6 +360,13 @@ here as well. Keys MUST appear in this order:
 . All other keys (except for `id` and `-orig-id`), in order specified
   in <<seq-attr>>
 
+[[enum-attr]]
+== Enum attributes
+
+* In the case of multiple enums with the same name it's usually
+  better to append the integer value rather than a sequence
+  number to its `id`.
+
```

Comment:
I had some problems understanding what you've meant here. Probably an example should help.

Reply:
https://github.com/kaitai-io/kaitai_struct_formats/blob/master/scientific/nt_mdt/nt_mdt.ksy#L632L634

Reply:
Yeah, that's a good idea. But we should describe it in more detail, with an example, and think of some other choices to be made here. For example, should it be `unknown31` or `unknown_31`? `unknown_31` or `unknown_1f` or `unknown0x1f`?

Reply:
>For example, should it be unknown31 or unknown_31? unknown_31 or unknown_1f or unknown0x1f?

No ```0x```, ```0o``` and ```0b``` prefixes, no underscore, the number part is the same as in the left part.